_includes/person_profile.html

```diff
@@ -45,45 +45,45 @@
     {% endif %}
 
     {% if person.homepage_url %}
-    <dt>Homepage</dt>
+    <dt>Sitio oficial</dt>
     <dd><a rel="nofollow" href="{{ person.homepage_url }}">{{ person.homepage_url }}</a></dd>
     {% endif %}
 
     {% if person.wikipedia_url %}
-    <dt>Wikipedia page</dt>
+    <dt>Página de Wikipedia</dt>
     <dd><a rel="nofollow" href="{{ person.wikipedia_url }}">{{ person.wikipedia_url }}</a></dd>
     {% endif %}
 
     {% if person.linkedin_url %}
-    <dt>LinkedIn page</dt>
+    <dt>Perfil de LinkedIn</dt>
     <dd><a rel="nofollow" href="{{ person.linkedin_url }}">{{ person.linkedin_url }}</a></dd>
     {% endif %}
 
   </dl>
 
 
-  <h2>Demographics:</h2>
+  <h2>Datos demográficos:</h2>
 
   <dl>
 
     {% if person.gender %}
-    <dt>Gender</dt>
+    <dt>Género</dt>
     <dd>{{ person.gender }}</dd>
     {% endif %}
 
     {% if person.birth_date %}
-    <dt>Date of birth</dt>
+    <dt>Fecha de nacimiento</dt>
     <dd>{{ person.birth_date }}</dd>
     {% endif %}
 
   </dl>
 
-  <h2>Photo Credit:</h2>
+  <h2>Crédito de la foto:</h2>
```

Comment:
I got this translation from the [YNR translations file for Costa Rica](https://github.com/mysociety/yournextrepresentative/blob/e515df71f76a4a153798208e5321119e3312a34c/locale/es_CR/LC_MESSAGES/django.po#L1782), so I think this should be OK.